src/expandableCalendar/style.ts

```diff
@@ -2,6 +2,7 @@ import {StyleSheet, Platform} from 'react-native';
 import * as defaultStyle from '../style';
 import {Theme} from '../types';
 import constants from '../commons/constants';
+import {HOURS_SIDEBAR_WIDTH} from '../timeline/style';
```

Comment:
I find it a little weird that ExpandableCalendar depends on the Timeline style. 
Is there any other way around it? Why does it need to know the hours sidebar width?

Reply:
Hmm, the header (dates and day names) should be flexed above the timeline with a left inset (we don't want it to be full width).. 
We can also pass this inset as a prop if you think it's better

Reply:
I think it's more correct to pass it as a prop